### DISCUSSION TITLE
Hello I lost access to my repository and email. It has over 100,000 commits and 12 contributors. The software is important to researchers even though it's not really important to any particular project. I really have to be in there maintaining it and it's not appropriate to have to make a duplicate site

### DISCUSSION THREAD

**Discussion by porschaa** [community/community#70755] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

So long story. I have changed my name and my life got crazy and due to really extenuated legal and medical circumstances outside of my control I lost access to my old email / Gmail account. In order to save my career I really have to get in to this account and update the copyrights and license with my new name. Unfortunately, I was a person who got Fame before money or power and the work is really important to researchers and it's not possible to rewrite. But there is also a lot of scandal involved where people who really do not have the background in mathematics to justify writing what they would like to call original work are taking my code. Which is fine in the end but it also has the effect of lowering the value of gnu's multiple precision library and it has the negative general effect of creating a new population of programmers who are doing math programming that used to only be done by people with PhD. I don't think that trying to work with Google Gmail to get into my old gmail account is going to be a productive use of time and my time is unfortunately limited. My time isn't valuable it's just I may not have much left and I have to do this. I can answer questions about the code that no one else can if needed and you guys have anyone that works with c run times or arbitrary precision math. Please help

**Comment by queenofcorgis** [comment, marked as answer]
Hi @alexandriapike ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
